trello2moment

```diff
@@ -133,7 +133,7 @@ function getBoard(boardPath) {
 
 async function getImages(momentName, jsonObj) {
   for ( let id in jsonObj ) {
-    if ( jsonObj[id].cover.idAttachment != null ) {
+    if ( jsonObj[id].cover.idAttachment != null || jsonObj[id].name === 'Triptych' ) {
```

Comment:
This makes no sense.  I expect that you use the label to make the trello2moment linked data file right?  So you should check for labels.  But, really, maybe remove this test entirely, and just fetch  all attachments?

Reply:
Added suggested fix and recommited/pushed.